drivers/serial/Kconfig.ns16550

```diff
@@ -30,17 +30,22 @@ config UART_NS16550_DRV_CMD
 
 choice
 	prompt "UART type"
```

Comment:
Since we have renamed the config to `UART_NS16550_VARIANT_`, updating the prompt would be nice:

```suggestion
	prompt "UART variant"
```

Reply:
Done.